kuma/javascript/src/logo.jsx

```diff
@@ -0,0 +1,20 @@
+//@flow
+import * as React from 'react';
+
+export default function(props : {|url: string|}) : React.Node {
+    return (
+        <a css={{margin:0}} className="logo" href={props.url}>
+          MDN Web Docs
+        </a>
+    );
+
+    /*
+      Or, to get the ducks back:
```

Comment:
I'll miss the ducks 🦆

Reply:
For reasons I did not figure out, the inilne CSS styles I was using to cover the dino head with the duck do not work when I try to duplicate them in react. Since I failed to get the ducks back, I just removed the entire comment.